ingestion/src/metadata/ingestion/source/database/datalake.py

```diff
@@ -176,6 +178,30 @@ def get_database_schema_names(self) -> Iterable[str]:
             else:
                 yield from self.fetch_s3_bucket_names()
 
+        if isinstance(self.service_connection.configSource, AzureDatalakeConfig):
+            yield from self.get_container_names()
+
+    def get_container_names(self) -> Iterable[str]:
+        schema_names = self.client.list_containers(name_starts_with="")
```

Comment:
@NiharDoshi99 what is name_starts_with="" will do?

Reply:
So it returns all the container name start with (whatever is passed), if nothing passed it will return all the containers.
`name_start_with` is a required argument need to be passed